tool/teleport/common/teleport.go

```diff
@@ -125,6 +125,8 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
 	start.Flag("config",
 		fmt.Sprintf("Path to a configuration file [%v]", defaults.ConfigFilePath)).
 		Short('c').ExistingFileVar(&ccf.ConfigFile)
+	start.Flag("apply-on-startup",
+		"Path to a YAML file containing resources to apply on startup. Works on initialized clusters, unlike --bootstrap. Only supports the following types: token.").ExistingFileVar(&ccf.ApplyOnStartupFile)
```

Comment:
Nit: Use corresponding "kind" constants instead of hardcoding "token" string.

Reply:
Also, this help string should probably mention that if provided, the file is expected not to be empty. Same for the bootstrap flag.